slice.go

```diff
@@ -693,3 +693,93 @@ func Splice[T any, Slice ~[]T](collection Slice, i int, elements ...T) Slice {
 
 	return append(append(append(output, collection[:i]...), elements...), collection[i:]...)
 }
+
+// Pull removes all given values from splice using SameValueZero for equality comparisons.
+func Pull[T comparable, Slice ~[]T](collection Slice, elements ...T) Slice {
+	output := make(Slice, 0, len(collection))
+
+	for _, item := range collection {
+		shouldRemove := false
+		for _, element := range elements {
+			if item == element {
+				shouldRemove = true
+				break
+			}
+		}
+
+		if !shouldRemove {
+			output = append(output, item)
+		}
+	}
+
+	return output
+}
+
+// This method is like Pull except that it accepts an array of values to remove
+func PullAll[T comparable, Slice ~[]T](collection Slice, elements []T) Slice {
```

Comment:
```suggestion
func PullAll[T comparable, Slice ~[]T](collection Slice, 
...elements T) Slice {
```
I would use a variadic, but then PullAll would be enough, no need for Pull.

So I would remove Pull func, and rename PullAll to Pull